From 362c0ece45a0bee8aef0a55ac6a64d6eba0b7d24 Mon Sep 17 00:00:00 2001
From: Sameer Mulla <sameer.mulla@lge.com>
Date: Thu, 22 Apr 2021 21:24:49 +0530
Subject: [PATCH] Avoid mesh cache data loading from storage path

:Release Notes:
Avoided mesh cache data loading from storage path

:Detailed Notes:
Disabling node_load_from_storage method call

:Testing Performed:
Built and tested

:QA Notes:
NA

:Issues Addressed:
[PLAT-136700] Analyze stability issue for provision

Upstream-Status: Inappropriate[webos specific]
---
 mesh/mesh.c | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/mesh/mesh.c b/mesh/mesh.c
index bc17037..4e98529 100644
--- a/mesh/mesh.c
+++ b/mesh/mesh.c
@@ -277,15 +277,17 @@ bool mesh_init(const char *config_dir, const char *mesh_conf_fname,
 
 	storage_dir = config_dir ? config_dir : MESH_STORAGEDIR;
 
-	l_info("Loading node configuration from %s", storage_dir);
+	l_info("Avoid Loading node configuration from %s", storage_dir);
 
 	if (!mesh_conf_fname)
 		mesh_conf_fname = CONFIGDIR "/mesh-main.conf";
 
 	parse_settings(mesh_conf_fname);
 
-	if (!node_load_from_storage(storage_dir))
-		return false;
+	/*if (!node_load_from_storage(storage_dir))
+		return false;*/
+
+	create_dir(storage_dir);
 
 	req = l_new(struct mesh_init_request, 1);
 	req->cb = cb;
-- 
2.26.0

